### ISSUE TITLE
Not streaming or pushing commits

### ISSUE THREAD

**Issue by pandinocoder** [mray2014/DayDreamEngine#2]
We are repeating the past (see #1 from two years ago).

We really need more streams, and code actually needs to be pushed.

We've seen changes happen on stream (like a week ago we saw a line or two get written) but those changes aren't pushed? That needs to be fixed.

**Comment by TreePuncher** [comment]
I agree. It's very disappoiting not having access to the latest version of the engine. I need the latest versions for my own projects!
